server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/ACVRDownloadByCounty.java

```diff
@@ -50,7 +55,7 @@ public EndpointType endpointType() {
    */
   @Override
   public String endpointName() {
-    return "/acvr/:counties";
+    return "/acvr/county/:counties";
```

Comment:
OK. Well, I'll change the endpoint for the get, anyway (in all the places where lists of county names appear - by the way, they are county names, not county IDs, in the current implementations).